src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/configuration/KafkaConsumerConfig.java

```diff
@@ -110,6 +111,22 @@
         return factory;
     }
 
+    /**
+     * Creates a concurrent kafka listener container factory.
+     * For each message a kafka listener receives this factory will create a container to process the message.
+     * Filter strategy is filtering out messages with timestamps after the app started
+     *
+     * @return A ConcurrentKafkaListenerContainerFactory
+     */
+    private ConcurrentKafkaListenerContainerFactory<String, ChipsRestInterfacesSend> getNewErrorContainerFactory() {
+        var currentTime = Instant.now().toEpochMilli();
+        ConcurrentKafkaListenerContainerFactory<String, ChipsRestInterfacesSend> factory =
+                new ConcurrentKafkaListenerContainerFactory<>();
+        factory.setConsumerFactory(newMainConsumerFactory());
+        factory.setRecordFilterStrategy(consumerRecord -> currentTime < consumerRecord.timestamp());
+        return factory;
```

Comment:
I would also add factory.setAckDiscarded(false); 
to ensure discarded messages don't have their offsets committed.